extensions/wikia/CuratedContent/maintenance/SpecialPromoteToCuratedContent.php

```diff
@@ -156,9 +158,9 @@ private function uploadImageToLocalWiki( $imageName, $imageUrl ) {
 		$uploadOptions->name = $imageName;
 		$uploadOptions->comment = wfMessage( 'wikiacuratedcontent-image-upload-comment' )->inContentLanguage()->escaped();
 		$uploadOptions->description = wfMessage( 'wikiacuratedcontent-image-upload-description' )->inContentLanguage()->escaped();
-		$wikiaBotUser = User::newFromName( 'WikiaBot' );
+		$botUser = User::newFromName( 'FandomBot' );
 
-		$result = ImagesService::uploadImageFromUrl( $imageUrl, $uploadOptions, $wikiaBotUser );
+		$result = ImagesService::uploadImageFromUrl( $imageUrl, $uploadOptions, $botUser );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "$result". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)